File: src/main/java/at/hannibal2/skyhanni/utils/jsonobjects/ArmorDropsJson.java
package at.hannibal2.skyhanni.utils.jsonobjects;

import com.google.gson.annotations.Expose;

import java.util.List;
import java.util.Map;

public class ArmorDropsJson {
    @Expose
    public Map<String, DropInfo> special_crops;

    public static class DropInfo {
        @Expose
        public String armor_type;

        @Expose
        public List<Double> chance;
    }
}

File: src/main/java/at/hannibal2/skyhanni/data/jsonobjects/repo/ArmorDropsJson.kt
package at.hannibal2.skyhanni.data.jsonobjects.repo

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class ArmorDropsJson(
    @Expose @SerializedName("special_crops") val specialCrops: Map<String, ArmorDropInfo>
)

data class ArmorDropInfo(
    @Expose @SerializedName("armor_type") val armorType: String,
    @Expose val chance: List<Double>
)
